scripts/mixins/follow.lua

```diff
@@ -0,0 +1,41 @@
+require('scripts/globals/mixins')
```

Comment:
Not sure if we need these requires anymore (maybe check with clay)

But we 100% need a header in here saying what this mixin is, and how to use it, and maybe why you'd use this and not the raw :follow/:unfollow bindings